docs/src/appendix/snforge/test.md

```diff
@@ -55,11 +55,6 @@ Control when colored output is used. Valid values:
 ## `--save-trace-data`
 Saves execution traces of test cases which have passed and are not fuzz tests to files. Traces can be used for profiling purposes.
 
-
-## `--use_scarb_collector`
-
-Uses Scarb to find and compile tests. Requires at least Scarb nightly-2023-12-04
-
 > ⚠️ **Warning**
 > 
 > This is an experimental feature. Some functionalities might not work.
```

Comment:
Shouldn't this be removed as well?

Reply:
Removed in [b0243bc](https://github.com/foundry-rs/starknet-foundry/pull/1511/commits/b0243bc40af6f111c4547d2f9dc5d5252ab0f251)